packages/react/select/src/Select.tsx

```diff
@@ -839,18 +737,286 @@ const SelectContentImpl = React.forwardRef<SelectContentImplElement, SelectConte
                   }
                 })}
               />
-            </FocusScope>
-          </div>
+            </DismissableLayer>
+          </FocusScope>
         </RemoveScroll>
       </SelectContentProvider>
     );
   }
 );
 
+SelectContentImpl.displayName = CONTENT_IMPL_NAME;
+
+/* -------------------------------------------------------------------------------------------------
+ * SelectPopperAbove
+ * -----------------------------------------------------------------------------------------------*/
+
+const POPPER_ABOVE_NAME = 'SelectPopperAbove';
+
+type SelectPopperAboveElement = React.ElementRef<typeof Primitive.div>;
+interface SelectPopperAboveProps extends PrimitiveDivProps, SelectPopperPrivateProps {}
+
+const SelectPopperAbove = React.forwardRef<SelectPopperAboveElement, SelectPopperAboveProps>(
+  (props: ScopedProps<SelectPopperAboveProps>, forwardedRef) => {
+    const { __scopeSelect, onPlaced, ...popperProps } = props;
+    const context = useSelectContext(CONTENT_NAME, __scopeSelect);
+    const contentContext = useSelectContentContext(CONTENT_NAME, __scopeSelect);
```

Comment:
```suggestion
    const context = useSelectContext(POPPER_ABOVE_NAME, __scopeSelect);
    const contentContext = useSelectContentContext(POPPER_ABOVE_NAME, __scopeSelect);
```

Reply:
That's on purpose as it's used for the error message users see, and for them the public part is `Content`, not the specific implementations.

Reply:
Fair